jha/src/views/enrolment/ChildInfoPage.vue

```diff
@@ -769,34 +201,19 @@ export default {
       });
       this.children = [...children];
     },
-    citizenshipSupportDocumentOptions(child) {
-      let options;
-
-      if (child.status === StatusInCanada.PermanentResident) {
-        options = selectOptionsPermanentResidentSupportDocuments;
-      } else if (child.status === StatusInCanada.TemporaryResident){
-        if (child.statusReason === CanadianStatusReasons.WorkingInBC) {
-          options = selectOptionWorkPermitSupportDocument;
-        } else if (child.statusReason === CanadianStatusReasons.StudyingInBC) {
-          options = selectOptionStudyPermitSupportDocument;
-        } else if (child.statusReason === CanadianStatusReasons.ReligiousWorker) {
-          options = selectOptionReligiousWorkSupportDocument;
-        } else if (child.statusReason === CanadianStatusReasons.Diplomat) {
-          options = selectOptionDiplomaticFoilSupportDocument;
-        } else if (child.statusReason === CanadianStatusReasons.Visiting) {
-          options = selectOptionVisitorVisaSupportDocument;
-        }
-      } else {
-        options = selectOptionsCitizenshipSupportDocuments;
+    setChildRef(el) {
```

Comment:
What does this function do? I don't see it being called anywhere in this PR.

Reply:
Removed. Sorry that was a leftover method from a different approach I tried